docs/source/using/troubleshooting.adoc

```diff
@@ -314,3 +314,28 @@ You can configure the URL by running:
 $ minishift config set check-network-http-host <URL>
 ----
 
+[[github-api-rate-limit-exceeded]]
+== Github API rate limit exceeded
+
+When you run `minishift start` or `minishift update`, it could make a request to Github to get the latest tarball of Minishift or Openshift client.
+Sometimes, during these requests, you may receive a 403 forbidden status from Github if your network has reached API rate limit and the command does not succeed, like the one below:
```

Comment:
* may > might
* Github > GitHub (search and replace)
* remove "has"
* API rate > the API rate
* like the one below > for example